.ci.yaml

```diff
@@ -6,6 +6,7 @@
 # More information at:
 #  * https://github.com/flutter/cocoon/blob/main/CI_YAML.md
 enabled_branches:
+  - flutter-2.8-candidate.10
```

Comment:
Is this still needed? The `  - flutter-\d+\.\d+-candidate\.\d+` should remove the need for it

Reply:
Probably not,  @christopherfujino is this something the conductor tool is adding automatically?